packages/store/addon/-private/system/schema-definition-service.ts

```diff
@@ -4,19 +4,8 @@ import { get } from '@ember/object';
 import { getOwner } from '@ember/application';
 import normalizeModelName from './normalize-model-name';
 import { RelationshipsSchema, AttributesSchema } from '../ts-interfaces/record-data-schemas';
-import require from 'require';
 import CoreStore from './core-store';
-import { HAS_MODEL_PACKAGE } from '@ember-data/private-build-infra';
-
-type Model = import('@ember-data/model').default;
-
-let _Model;
-function getModel() {
-  if (HAS_MODEL_PACKAGE) {
-    _Model = _Model || require('@ember-data/model').default;
-  }
-  return _Model;
-}
+import { RecordInstance } from '../ts-interfaces/record-instance';
```

Comment:
`RecordInstance` is for the instantiated record instances, I believe based on usage below you are in need of `ModelSchema` or `DSModelSchema`